eap-latest-setup.md

```diff
@@ -1697,9 +1697,79 @@ directory with:
 
     mkdir -p /mnt/registry
 
+### Creating user account for registry
+Each pod runs under particular service account. Three such accounts are created for you
+by default in each project:
+
+    os get serviceaccounts
+    NAME       SECRETS
+    builder    2
+    default    2
+    deployer   2
+
+`builder` service account is required by builder pods and `deployer` by
+deployment pods. The `default` is used for all the other pods unless
+overridden.
+
+Each service account can be given security context constraints (SCC). They
+allow to specify SELinux context for pods, limit users used to run them etc.
+To list them, type:
+
+    oc get scc
+    NAME         PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER
+    privileged   true      []        true      RunAsAny    RunAsAny
+    restricted   false     []        false     MustRunAs   MustRunAsRange
+
+By default, newly created pods are *restricted* which protects users from compromising
+the system. However, restricted pod does not allow you to mount host directory. The
+registry pod needs be run under *privileged* service account. Since all currently
+existing have already their use, let's set up a new one as `root`:
+
+    oc create -f - <<EOF
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: registry
+    EOF
+
+Next step is to make it *privileged*. This is achieved by adding service
+account's user name to a list of users of *privileged* SCC. Such a name looks
+like this:
+
+    system:serviceaccount:<namespace>:<name>
+
+Where `<namespace>` is `default` because we set up registry in the *default*
+namespace and `<name>` is whatever we used in previous `oc create` command.
+
+Type following command and add the user name:
+
+    oc edit scc priveliged
```

Comment:
@pweil- I don't remember what led to the host mount requiring a privileged container :frowning: It would be interesting to see what happens if you remove that. I'm guessing you'd maybe run into some of the stuff @pmorie is working on w.r.t. volumes & security.